modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/helper/SaveFormInstanceMVCCommandHelper.java

```diff
@@ -105,7 +106,24 @@ public DDMFormInstance saveFormInstance(
 			PortletRequest portletRequest, PortletResponse portletResponse)
 		throws Exception {
 
-		return saveFormInstance(portletRequest, portletResponse, false);
+		HttpServletRequest httpServletRequest = _portal.getHttpServletRequest(
+			portletRequest);
+
+		try {
+			AuthTokenUtil.checkCSRFToken(
+				httpServletRequest,
+				SaveFormInstanceMVCCommandHelper.class.getName());
+
+			return saveFormInstance(portletRequest, portletResponse, false);
+		}
+		catch (PortalException portalException) {
+			_log.error(
+				_language.get(
+					httpServletRequest, "csrf-token-is-missing-or-invalid"),
+				portalException);
```

Comment:
@rijema We don't need to use a language key for logs. We can do the same as it was done [here](https://github.com/liferay/liferay-portal/blob/4f00498af95271972fd93e32735752a557648c3c/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/ValidateCSRFTokenMVCResourceCommand.java#L68-L70).

Reply:
Done. Thank you.